### ISSUE TITLE
[Bug] Solr not indexing uploaded pdf documents

### ISSUE THREAD

**Issue by jkim-ru** [dibbs-vdc/ccql#14]
Not sure when this started, but the only way for documents to be indexed is by running from the console:

ActiveFedora::Base.reindex_everything

**Comment by jkim-ru** [comment]
There seems to be a difference in Solr when I compare a newly created FileSEt and an existing FileSet viewed via the Solr Admin UI after a call to ``ActiveFedora::Base.reindex_everything``.

Here's how I'm viewing all of my file sets:

```
http://localhost:8983/solr/hydra-development/select?fq=has_model_ssim:%22FileSet%22&indent=on&q=*:*&wt=json
```

The newly created file set is missing the following fields:

```
        "file_format_tesim":["pdf (Portable Document Format)"],
        "file_size_lts":1373814,
        "mime_type_ssi":"application/pdf",
        "digest_ssim":["urn:sha1:c68fc341e4cabead8e523cf8340c250d376cbb73"],
        "page_count_tesim":["23"],
        "file_title_tesim":["Thank your for completing the Innovation Survey.",
          "Local Disk"],
        "original_checksum_tesim":["8b5c0cc9db236cc61b9891ea59f8cd34"],
 ```

**Comment by jkim-ru** [comment]
These all look like they're all set by the ``FileSetIndexer`` (https://github.com/samvera/hyrax/blob/v1.0.4/app/indexers/hyrax/file_set_indexer.rb).

**Comment by jkim-ru** [comment]
Related topic on samvera-tech GG: https://groups.google.com/forum/#!topic/samvera-tech/ogBCi1Gf6B8

**Comment by jkim-ru** [comment]
I ended up going into my generated works controller and adding the following two lines to both my create and update methods:

curation_concern.reload #Get extracted_text loaded properly for each of the curation_concern.members
curation_concern.members.each{|member| member.update_index} # Reindex Solr

I put more in the samvera GG link above, but here's one relevant section cut and pasted from it.

What I was finding was that, in FileActorSet.attach_file_to_work, the call to work.reload (link below) wasn't loading the extracted_text. So, when the solr index got updated a few lines later, it wasn't indexed properly.

 https://github.com/samvera/hyrax/blob/4afb5524a6c259d1312f88c157431e77350fcb5f/app/actors/hyrax/actors/file_set_actor.rb#L47

I feel like the *real* solution is that the extracted_text needs to be saved somewhere before we get to the point above, but I don't know where to do that yet.  Even if I did find a good place for modification, I'd most likely need to open an issue with the Hyrax folks and get it changed with them.
